projectdata/dependencies.json

```diff
@@ -1,15 +1,17 @@
 [
-    {   "name": "ohNet",
-        "type": "openhome",
+    {
+        "name": "ohNet",
+        "type": "external",
         "platform-specific": true,
-        "version": "1.31.4245",
-        "archive-suffix": "-${debugmode}",
-        "main-dir": "${dest}${name}-${platform}-${debugmode}",
+        "version": "1.32.4350",
+        "cross-check": false,
+        "archive-filename": "${name}-${platform}-${debugmode}.tar.gz",
+        "archive-path": "https://github.com/KGrzeg/ohNet/releases/download/ohNet_${version}/${archive-filename}",
```

Comment:
`merakiacoustic` instead of `KGrzeg` would be more suiting ;)

Reply:
We need to bury merakiacoustic/ohNet, then I can transfer the repo.